packages/app/pages/EconomicEvent/CreateEconomicEvent.tsx

```diff
@@ -0,0 +1,182 @@
+import * as React from "react";
```

Comment:
Missing some documentation at the start of the file. (author, date)

Reply:
Added header comments